docs/concepts/what-is-multi-cluster-replication.md

```diff
@@ -339,10 +339,10 @@ Due to the nature of Multi-cluster Replication (for example, Workflow Execution
         |                          |                          |
         |                          | Run 1 Replication Events |
         |                          | -----------------------> |
-        |                          |                          |
-| ------------- |          | ------------- |          | ------------- |
-|   Cluster A   |          | Network Layer |          |   Cluster B   |
-| ------------- |          | ------------- |          | ------------- |
+        |     |  |
+        | --- || ------------- |          | ------------- |
+| Cluster A |  | Network Layer |  | Cluster B |
+| --------- || ------------- |          | ------------- |
 ```
 
 Since Run 2 appears in Cluster B first, Run 1 cannot be replicated as "runnable" due to the rule `at most one Run open` (see above), thus the "zombie" Workflow Execution state is introduced.
```

Comment:
```suggestion
Because Run 2 appears in Cluster B first, Run 1 cannot be replicated as "runnable" due to the rule `at most one Run open` (see above); thus the "zombie" Workflow Execution state is introduced.
```